modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java

```diff
@@ -269,6 +281,52 @@ protected UserGroup testPutUserGroupByExternalReferenceCode_addUserGroup()
 		return _postUserGroup();
 	}
 
+	private void _assertContains(
+		long creatorId, List<UserGroup> items, long totalCount) {
+
+		long count = 0;
+		boolean found = false;
+
+		for (UserGroup item : items) {
+			Creator creator = item.getCreator();
+
+			if (creator.getId() == creatorId) {
+				count++;
+			}
+		}
+
+		if (count >= totalCount) {
+			found = true;
+		}
+
+		Assert.assertTrue(
+			"UserGroup with creator ID " + creatorId + " not found", found);
+	}
+
+	private void _assertContains(User user, List<UserGroup> items) {
+		boolean found = false;
+
+		for (UserGroup item : items) {
+			for (UserAccountBrief userAccountBrief :
+					item.getUserAccountBriefs()) {
+
+				if (userAccountBrief.getId() == user.getUserId()) {
+					found = true;
+
+					break;
+				}
+			}
+
+			if (found) {
+				break;
+			}
+		}
+
+		Assert.assertTrue(
+			"UserAccountBrief with ID " + user.getUserId() + " not found",
+			found);
+	}
```

Comment:
I think these are not useful because can be easily replaced with some ListUtil.exists / contains or something like that